docs/runbooks/KubeMacPoolDuplicateMacsFound.md

```diff
@@ -0,0 +1,31 @@
+# KubeMacPoolDuplicateMacsFound
+
+## Meaning
+
+KubeMacPool is responsible to allocate mac addresses and avoid mac address conflicts.
+Upon starting KubeMacPool, it scan the current VM instances and updates it's database.
+In case it find duplicates it increase a counter of mac duplicate, which results in this alert.
+Duplicates will not be updated in the database but do exists until solved.
+
+## Impact
+
+Having a duplicate mac address can break the requirement that each interface on the same LAN,
+should have a unique mac address.
+
+## Diagnosis
+
+- Check the logs of kubemacpool-mac-controller, find the lines that say
```

Comment:
I would be more specific.. don't expect the sys-admin to know what's KMP's namespace
```
- Get kubemacpool-mac-controller pod namespace + command example
- Extract the conflicting VMs from kubemacpool pod logs, from which you will have 1, 2 and 3: + command example
```